ga4gh/datarepo.py

```diff
@@ -400,33 +412,50 @@ def _readSystemTable(self, cursor):
         self._creationTimeStamp = config["creationTimeStamp"]
 
     def _createOntologyTable(self, cursor):
+        # Right now we support a crude ontology term name-ID bidirectional
+        # map. However, in the future we will want to have better ontology
+        # support. Therefore we're not making the SQL schema too specific
+        # so that we can make this transition without needing backwards
+        # incompatible schema changes.
         sql = """
```

Comment:
As far as I can tell there are release numbers [here](http://www.sequenceontology.org/browser/obob.cgi). Perhaps an argument when adding via repo manager? Let's avoid adding anything else to this custom filetype, like version headers.